fdpa-resolutions.html

```diff
@@ -2,79 +2,21 @@
+  <meta name="referrer" content="origin">
   <style>
     #map-canvas { width:1100px; height:650px; }
-    .layer-wizard-search-label { font-family: sans-serif };
+    .layer-wizard-search-label { font-family: sans-serif }
+    .hidden { display:none; }
   </style>
-  <script type="text/javascript"
-    src="https://maps.google.com/maps/api/js?key=AIzaSyBxdPyKNXkcFbFPs5I_e-xqGCUPGVuyabs">
-  </script>
-  <script type="text/javascript">
-    var map;
-    var layer_0;
-    var layer_1;
-    function initialize() {
-      map = new google.maps.Map(document.getElementById('map-canvas'), {
-        center: new google.maps.LatLng(40.99058069667954, -77.62118299218751),
-        zoom: 8,
-        zoomControl: true,
-            zoomControlOptions: {
-            position: google.maps.ControlPosition.TOP_LEFT
-        },
-        fullscreenControl: true,
-        streetViewControl: false,
-        mapTypeControl: false
-      });
-      var style = [
-        {
-          featureType: 'all',
-          elementType: 'all',
-          stylers: [
-            { saturation: -85 }
-          ]
-        },
-        {
-          featureType: 'poi',
-          elementType: 'all',
-          stylers: [
-            { visibility: 'off' }
-          ]
-        }
-      ];
-      var styledMapType = new google.maps.StyledMapType(style, {
-        map: map,
-        name: 'Styled Map'
-      });
-      map.mapTypes.set('map-style', styledMapType);
-      map.setMapTypeId('map-style');
-      layer_0 = new google.maps.FusionTablesLayer({
-        query: {
-          select: "col12>>0",
-          from: "1emKZpIHeBPHfcWjdckvWJ_DWmgNrJYTRAMrmllzJ",
-          where: " col3>>0 = 'County'"
-        },
-        map: map,
-        styleId: 3,
-        templateId: 4
-      });
-      layer_1 = new google.maps.FusionTablesLayer({
-        query: {
-          select: "col16>>0",
-          from: "1emKZpIHeBPHfcWjdckvWJ_DWmgNrJYTRAMrmllzJ",
-          where: " col3>>0 in ('Borough', 'City', 'Town', 'Township (1st)', 'Township (2nd)', 'Organization') and col5>>0 in ('In work', 'Materials sent', 'Passed')"
-        },
-        map: map,
-        styleId: 4,
-        templateId: 5
-      });
-    }
-    google.maps.event.addDomListener(window, 'load', initialize);
-  </script>
   </head>
   <body>
     <h2>Fair Districts PA - Local Resolutions</h2>
-    <p>To date, more than 130 counties and municipalities across Pennsylvania have adopted a
+    <p id="fall-back-count">To date, more than 198 counties and municipalities across Pennsylvania have adopted a
         resolution in support of an indpendent citizens' redistricting commission.</p>
+    <p id="current-count" class="hidden">To date,
+        <span data-status="passed"></span> counties and municipalities across Pennsylvania have adopted
+        a resolution in support of an independent citizens' redistricting commission.
+        <span data-status="inWork"></span> more are in progress.</p>

```

Comment:
So here's a thing I forgot to explain... the yellow dots on the map are actually "In work" and "Materials sent" together.  It would be best for this number to match the number of yellow dots.  It looks like your API is set up to allow both queries, so can you just do a quick sum in the page here for injecting into the span?

Reply:
I'll change the fdpa-resolutions getResolutionsCount to accept an array in the status option, and it will send an OR in the sql query. That's how to get absolutely consistent numbers. Thanks for the clarification.